h/h_api/bulk_api/entry_point.py

```diff
@@ -89,3 +102,40 @@ def _commands_from_ndjson(lines):
 
             # Try catch JSON errors here
             yield CommandBuilder.from_data(data)
+
+    @staticmethod
+    def _string_to_lines(string):
+        """Split a string on new-lines.
+
+        :param string: A string to split
+        :return: A generator of strings containing single lines
+        """
+        return (line for line in string.strip().split("\n"))
```

Comment:
I'm not sure why a generator is needed here but :man_shrugging:

Reply:
Oops. I was going to strip the line on the front, but it's not necessary as JSON tolerates trailing whitespace.